utils/build/docker/java/jersey-grizzly2/src/main/java/com/datadoghq/jersey/MyResource.java

```diff
@@ -313,64 +317,47 @@ public String insecurePathTraversal(@FormParam("path") final String path) {
 
     @POST
     @Path("/iast/source/parameter/test")
-    public String sourceParameter(@FormParam("source") final String source) {
-        final Span span = GlobalTracer.get().activeSpan();
```

Comment:
I'm out of context, I guess removing the GlobalTracer makes sense....

Reply:
Yes it was there due to copy and paste